lib/exceptions/application_validator.js

```diff
@@ -82,6 +94,28 @@ function checkTestFrameworkValues(values) {
   });
 }
 
+function checkDatabaseTypeValue(value) {
+  if (!ApplicationOptions.databaseType[value]) {
+    throw new Error(`Unknown value '${value}' for option 'databaseType'.`);
+  }
+}
+
+function checkDevDatabaseTypeValue(value) {
+  if (
+    !ApplicationOptions.databaseType[value] &&
```

Comment:
Does this allow `sql` to be used as a devDatabaseType?  Same thing below for `checkProdDatabaseTypeValue`

Reply:
I know, it's not perfect yet. It must be improved.
We should check consistency, for example:
- databaseType=sql, devDatabaseType=mongodb, prodDatabaseType=mongodb -> should be forbidden
- databaseType=mongodb, devDatabaseType=mysql, prodDatabaseType=mysql -> should be forbidden too

Reply:
I'm doing it right now :)